public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -88,10 +88,11 @@ export function AnomalyResults(props: AnomalyResultsProps) {
       BREADCRUMBS.DETECTORS,
       { text: detector ? detector.name : '' },
     ]);
+    dispatch(getDetector(detectorId));
   }, []);
 
   const fetchDetector = async () => {
-    await dispatch(getDetector(detectorId));
+    dispatch(getDetector(detectorId));
```

Comment:
wondering why removing `await` here?

Reply:
Unnecessary, since dispatch() is just used to trigger a separate async action to update the redux store, and the fn itself is already async. Also, nothing is done after the action is done within this function (like catch errors, do something with the response, etc). VS code also complains about it being unnecessary / has no effect.

Once the action is done, the page will render with the updated `detector` since the redux store will be updated.

Reply:
```
VS code also complains about it being unnecessary / has no effect.
```
Hmm, why I didn't see that in my VS code. is it because of any good plugin?

Reply:
Not sure, don't have extensions specific to javascript/typescript/react. Just looks like this. Have also noticed it in many other places, just fixed here though.

![Screen Shot 2020-09-03 at 11 35 02 AM](https://user-images.githubusercontent.com/62119629/92153712-90b9c000-edd9-11ea-9d8f-2e0e9646378c.png)